package/yast2-network.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Fri Feb 23 09:56:48 UTC 2016 - knut.anderssen@suse.com
+
+- bsc#903190
+  - Recognized netcards with subclass 07 as InfiniBand (ib) which
+  now are not filtered out.
```

Comment:
Exactly, the fix was merged but i will continue with the clean up, just after i finish with yast/yast-snapper#40 and yast/yast-snapper#39 i'm working on it currently.